db/migrate/20181121145009_create_active_admin_comments.rb

```diff
@@ -0,0 +1,17 @@
+class CreateActiveAdminComments < ActiveRecord::Migration::Current
```

Comment:
I think you don't need this. Remember to rollback before deleting this file and set `config.comments = false` in active admin config.

Reply:
Alright. I included it because I wanted to know from you guys whether admin comments are necessary or not. I'll remove this.